### PR TITLE
Studio: Update WordPress.com logo

### LINKED ISSUE #620: Studio: Update WordPress.com logo
### Quick summary

The WordPress.com logo shown when a user is logged out is outdated and needs updating.

Image

### Steps to reproduce

1. Open Studio
2. Ensure you are not logged in to a WordPress.com account

### What you expected to happen

Should show the latest logo.

### What actually happened

The old WordPress.com logo is used.

### Impact

All

### Available workarounds?

There is no user impact

### Platform

_No response_

### Logs or notes

_No response_

### PR DESCRIPTION
**Pull request by matt-west**
Fixes #620 

## Proposed Changes

- Update the WordPress.com logo to the latest version.
- Update spacing and alignment in the header bar for new logo.

| Before | After |
|--------|--------|
| Screenshot 2024-10-24 at 14 24 32 | Screenshot 2024-10-24 at 14 24 22 | 

## Testing Instructions

- Open Studio
- Ensure you are logged out of WordPress.com

## Pre-merge Checklist

- [x] Have you checked for TypeScript, React or other console errors?

### CODE REVIEW

File: src/components/wordpress-logo.tsx [hunk 1]
```diff
@@ -1,21 +1,16 @@
...
+				fill-rule="evenodd"
```
Comment (katinthehatsite) [inline src/components/wordpress-logo.tsx]:
```suggestion
				fillRule="evenodd"
```

File: src/components/wordpress-logo.tsx [hunk 2]
```diff
@@ -1,21 +1,16 @@
...
+				clip-rule="evenodd"
```
Comment (katinthehatsite) [inline src/components/wordpress-logo.tsx]:
```suggestion
				clipRule="evenodd"
```

**Review by katinthehatsite — COMMENTED** [review]
The logo looks good to me. I added some suggestions for `fillRule` and `clipRule` - otherwise, we would get this error in the console:

Screenshot 2024-10-25 at 9 16 20 AM

**Comment by matt-west** [comment]
Thanks for catching that @katinthehatsite!

I’ve applied your suggestions and tested locally to verify that the console warnings are gone.

**Review by katinthehatsite — APPROVED** [review]

**Comment by katinthehatsite** [comment]
Sounds good @matt-west - I approved the PR!

**Comment by wojtekn** [comment]
@matt-west the new logo looks better, thanks for the fix. I've just noticed a slight layout shift between logged-in and logged-out states:

Screenshot 2024-10-25 at 09 53 27

**Comment by matt-west** [comment]
@wojtekn Good catch! I’ve removed the unnecessary vertical padding from the logo.

This should fix the layout shift between logged in/out states.

image

**Comment by wojtekn** [comment]
@matt-west looks great now!

**Review by wojtekn — APPROVED** [review]

**Pull request merged**